section2/exercises/methods.rb

```diff
@@ -15,17 +15,63 @@ def print_name(name)
   # YOUR CODE HERE
 end
 
-print_name("Albus Dumbledore")
+def print_name
+  p "Albus Dumbledore"
+end
+
+print_name
 
-# Write a method that takes in 2 numbers as arguments and prints 
+# Write a method that takes in 2 numbers as arguments and prints
```

Comment:
They wanted you to write a function and then call it three times. You declared three different functions. Also, your functions print the numbers that are passed in, but don't return the sum.

```ruby
def add(num1, num2)
     puts num1 + num2
end

add(3, 4)
# => 7

add(20, 5)
# => 25
```